src/routes/company/+page.svelte

```diff
@@ -29,560 +71,711 @@
.091C431.93 253.909 432.358 253.818 432.82 253.818C433.161 253.818 433.453 253.847 433.695 253.903C433.938 253.956 434.118 254.006 434.235 254.051L433.837 255.426C433.758 255.403 433.656 255.377 433.531 255.347C433.406 255.312 433.254 255.295 433.076 255.295C432.663 255.295 432.368 255.398 432.19 255.602C432.016 255.807 431.928 256.102 431.928 256.489V266H430.23ZM435.49 266V257.273H437.132V258.659H437.223C437.382 258.189 437.662 257.82 438.064 257.551C438.469 257.278 438.927 257.142 439.439 257.142C439.545 257.142 439.67 257.146 439.814 257.153C439.962 257.161 440.077 257.17 440.16 257.182V258.807C440.092 258.788 439.971 258.767 439.797 258.744C439.622 258.718 439.448 258.705 439.274 258.705C438.872 258.705 438.515 258.79 438.2 258.96C437.89 259.127 437.643 259.36 437.462 259.659C437.28 259.955 437.189 260.292 437.189 260.67V266H435.49ZM444.02 266.193C443.467 266.193 442.967 266.091 442.52 265.886C442.073 265.678 441.719 265.377 441.458 264.983C441.2 264.589 441.071 264.106 441.071 263.534C441.071 263.042 441.166 262.636 441.355 262.318C441.545 262 441.801 261.748 442.122 261.562C442.444 261.377 442.804 261.237 443.202 261.142C443.6 261.047 444.005 260.975 444.418 260.926C444.941 260.866 445.365 260.816 445.691 260.778C446.016 260.737 446.253 260.67 446.401 260.58C446.549 260.489 446.622 260.341 446.622 260.136V260.097C446.622 259.6 446.482 259.216 446.202 258.943C445.926 258.67 445.513 258.534 444.963 258.534C444.391 258.534 443.941 258.661 443.611 258.915C443.285 259.165 443.06 259.443 442.935 259.75L441.338 259.386C441.528 258.856 441.804 258.428 442.168 258.102C442.535 257.773 442.958 257.534 443.435 257.386C443.912 257.235 444.414 257.159 444.941 257.159C445.289 257.159 445.658 257.201 446.049 257.284C446.443 257.364 446.81 257.511 447.151 257.727C447.496 257.943 447.778 258.252 447.997 258.653C448.217 259.051 448.327 259.568 448.327 260.205V266H446.668V264.807H446.6C446.49 265.027 446.325 265.242 446.105 265.455C445.886 265.667 445.604 265.843 445.259 265.983C444.914 266.123 444.501 266.193 444.02 266.193ZM444.39 264.83C444.859 264.83 445.261 264.737 445.594 264.551C445.931 264.366 446.187 264.123 446.361 263.824C446.539 263.521 446.628 263.197 446.628 262.852V261.727C446.568 261.788 446.45 261.845 446.276 261.898C446.105 261.947 445.91 261.991 445.691 262.028C445.471 262.062 445.257 262.095 445.049 262.125C444.84 262.152 444.666 262.174 444.526 262.193C444.196 262.235 443.895 262.305 443.622 262.403C443.354 262.502 443.138 262.644 442.975 262.83C442.816 263.011 442.736 263.254 442.736 263.557C442.736 263.977 442.891 264.295 443.202 264.511C443.513 264.723 443.908 264.83 444.39 264.83ZM457.058 259.403L455.518 259.676C455.453 259.479 455.351 259.292 455.211 259.114C455.075 258.936 454.889 258.79 454.654 258.676C454.419 258.562 454.126 258.506 453.773 258.506C453.292 258.506 452.891 258.614 452.569 258.83C452.247 259.042 452.086 259.316 452.086 259.653C452.086 259.945 452.194 260.18 452.41 260.358C452.626 260.536 452.974 260.682 453.455 260.7
```

Comment:
```suggestion
                                                <div class="web-user-box-username web-sub-body-400">CTO @ mySHOEFITTER</div>
```
![Screenshot 2024-04-15 at 11 44 05 PM](https://github.com/appwrite/website/assets/20852629/17ed5ef9-3f29-43ae-93eb-193b3ca5cac3)

I believe its more common and relevant to use the `@` instead of `//` . Please check this option with the design team as well